src/Postmark/Model/Suppressions/PostmarkSuppression.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Postmark.Model.Suppressions
+{
+    /// <summary>
+    /// Model representing a Suppressed recipient.
+    /// For more information about the Suppressions API and possible options, please visit:
+    /// https://postmarkapp.com/developer/api/suppressions-api
+    /// </summary>
+    public class PostmarkSuppression
+    {
+        /// <summary>
+        /// Email address of the suppressed recipient.
+        /// </summary>
+        public string EmailAddress { get; set; }
+
+        /// <summary>
+        /// Reason why this recipient was suppressed. E.g.: ManualSuppression, HardBounce, SpamComplaint.
+        /// </summary>
+        [JsonConverter(typeof(StringEnumConverter))]
+        public PostmarkSuppressionReason SuppressionReason { get; set; }
```

Comment:
I’m not sure we should use an enumeration here. If we end up adding new reasons this will break the existing integrations.

Reply:
That's right. I was pondering this question when I implemented it and my thinking was that adding new reasons/origins was really rare. But it's true that adding one could require people to update their libraries or break their stuff. I updated those to `strings` and documented this better instead.